docs/modules/vertiq_81xx_family.rst

```diff
@@ -143,7 +144,10 @@ Next Steps
 ********************************************
 Now that you have successfully wired your module for use, feel free to complete the :ref:`Getting Started Guide <81_getting_started>` for your module and its firmware style.
 
-81-08 Generation 2 (G2) Hardware Setup Walkthrough
+.. note::
+        While the images below show an 81-08 size module, these setup instructions are also valid for an 81-17 size module.
```

Comment:
I didn't add new setup stuff for the 81-17, the 81-08 stuff is applicable. I just put this note to indicate that